packages/rum-core/src/domain/rumEventsCollection/action/trackActions.ts

```diff
@@ -138,14 +139,27 @@ class PendingAutoAction {
   }
 
   complete(endTime: TimeStamp) {
+    const actionDuration = elapsed(this.startClocks.timeStamp, endTime)
+    if (actionDuration < 0) {
+      addMonitoringMessage('auto action with negative loading time', {
+        debug: {
+          actionDuration,
+          type: this.type,
+          name: this.name,
+          startClocks: this.startClocks,
+          endTime,
```

Comment:
I'm not sure how this information will help you to understand what's going on. The `startClocks.timeStamp` will always be greater than `endTime`, which is the issue I know, but what could we deduce from that?

Maybe storing the "clock state" in `endTime` instead of a timestamp alone could help, we could maybe observe that comparing timestamps produces a negative time, but comparing relative times produces a positive time.

Reply:
Noted, thanks for the advice. I'll add the end clock state.